storage/tests/system.py

```diff
@@ -245,6 +245,27 @@ def test_copy_existing_file(self):
         self.assertEqual(base_contents, copied_contents)
 
 
+class TestUnicode(unittest.TestCase):
+    @unittest.skipIf(six.PY2, 'Unicode URLs only work on Python 3.')
```

Comment:
Change something. See https://gist.github.com/dhermes/cc2d8e42ccd8825efd547152816da899

It seems if we convert to `bytes` first than all will be well?